front/assets/translations/fr_FR.json

```
@@ -4,11 +4,18 @@
   },
   "common": {
     "cancel": "Annuler",
-    "delete": "Supprimer"
+    "delete": "Supprimer",
+    "confirm": "Confirmer"
   },
   "jobOffer": {
     "info": {
+      "jobOfferList": "Liste des offres d'emploi",
       "jobOffer": "Offre d'emploi"
+    },
+    "apply": {
+      "apply": "Postuler",
+      "applyTitle": "Confirmation de candidature",
+      "applyConfirm": "Do you want to apply to this job"
```

You forgot to translate this.

```suggestion
      "applyConfirm": "Voulez-vous candidater à cette offre ?"
```